input/resources/au-core-requester.xml

```diff
@@ -2850,6 +2927,49 @@
 </div>
 </div>
 </div>
+<div class="panel panel-default">
+<div class="panel-heading">
+<h4 id="RelatedPerson1-1" class="panel-title">
+<span style="float: right;">Resource Conformance: SHOULD</span>RelatedPerson</h4>
+</div>
+<div class="panel-body">
+<div class="container">
+<div class="row">
+<div class="col-lg-6">
+<span class="lead">Base System Profile</span>
+<br/>
+<a href="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson">http://hl7.org.au/fhir/StructureDefinition/au-relatedperson</a>
+</div>
+<div class="col-lg-3">
+<span class="lead">Profile Conformance</span>
```

Comment:
The Profile Conformance is SHALL. Apply this here and in the structure.

Reply:
Applied profile conformance to narrative, bolded SHALL in documentation for narrative and structure.